webdriver/commands/AcceptAlert.json

```diff
@@ -5,25 +5,21 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/WebDriver/Commands/AcceptAlert",
           "support": {
-            "webview_android": {
-              "version_added": false,
-              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the capabiliites when running the command."
-            },
             "chrome": {
-              "version_added": false,
-              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the capabiliites when running the command."
+              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the capabiliites when running the command.",
```

Comment:
I assume it wasn't intentional that the child properties were reordered as well?

Reply:
[It wasn't](https://github.com/mdn/browser-compat-data/pull/2899#discussion_r222039202), which is another reason for me to switch over to a new JSON editor.  So many issues... X3  Will fix this up to revert those changes soon!